docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/button/button.twig

```diff
@@ -70,10 +70,16 @@
   {% endif %}
 {% endset %}
 
+{% if is_new_window %}
+  {% set is_new_window_text %}
+    <span class="civictheme-button--is-new-window">Opens in a new tab/window.</span>
```

Comment:
same as in https://github.com/salsadigitalauorg/civictheme_source/pull/638

Please use the `civictheme-visually-hidden` class and remove the custom class.

Reply:
also, please remove the dot